### ISSUE TITLE
HTTPD instrumentation: Safe modification of httpd.conf 

### ISSUE THREAD

**Issue by CostisC** [open-telemetry/opentelemetry-operator#3401]
### Component(s)

auto-instrumentation

## Description
The `Include /usr/local/apache2/conf/opentemetry_agent.conf` may be appended on the same line with the last httpd.conf statement, if there is not a blank last line in httpd.conf. This results in a corrupted httpd configuration, and the service will not start.

## Steps to Reproduce
Change httpd.conf to not have an blank last line, and then restart the instrumented httpd pod. It fails to start up.

## Expected Result
The `Include /usr/local/apache2/conf/opentemetry_agent.conf`  statement should be appended as a separate last line, in httpd.conf settings. 

## Actual Result
E.g. the last lines of the new httpd.conf become:  
```
<IfModule http2_module>
   <!-- omitted -->
</IfModule>Include /usr/local/apache2/conf/opentemetry_agent.conf
```

### Kubernetes Version

v1.30.5-gke.1014001

### Operator version

v0.107.0

### Collector version

v0.14.0

**Comment by CostisC** [comment]
Suggested fix in PR #3395

**Comment by jaronoff97** [comment]
closed by #3395
